src/routes/App.re

```diff
@@ -96,8 +96,7 @@ let make = () => {
         switch (url.path) {
         | [network, entity, id] => goToPage(network, entity, id)
         | [network] => goToNetwork(network)
-        | _ =>
-          goToMainPage(configs[state.selectedConfig].network);
+        | _ => goToMainPage(configs[state.selectedConfig].network)
```

Comment:
I see that the trailing semicolon is missing, might be relevant?

Reply:
Will fix and redeploy, surprised it didn't throw up an error.